tests/migration/storage.go

```diff
@@ -0,0 +1,152 @@
+package migration
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/nspcc-dev/neo-go/pkg/util"
+	"github.com/stretchr/testify/require"
+)
+
+// Options groups optional parameters for TestRemoteStorageMigration.
```

Comment:
It tries to cover every possible case, but probably we can simplify this. The way I'd like to see these tests is:
 * get previously stored data, init storage with it
 * for every test create a secondary memcache (to keep initial one clean between tests)
 * initialize Blockchain with it
 * perform action
 * check result
 * drop this instance and move to the new test

Current scheme limits `Options` to a single successful run, but we might as well have a number of them (even if they're based on the same data, it can be tweaked in some ways by test).